docs/guides/introduction/about.md

```diff
@@ -1 +1,23 @@
-# About Daffodil
+# Introduction
+
+## What is Daffodil?
+
+Daffodil is a Typescript framework for building headless ecommerce stores. It is built on top of Angular and provides a suite of composable packages that allow you to build complex ecommerce stores.
+
+These packages provide tools and architectures that allow you to quickly build storefronts that are:
+
+1. Performant
+2. Reliable
+3. Interoperable
+4. Maintainable
+5. Accessible
+
+## The Framework
+
+Daffodil is built with flexibility and scalability in mind. Here are a few features that we believe make Daffodil special:
+
+- **Drivers**: [Drivers](/docs/guides/drivers.md) are Daffodil's lifeblood. The drivers enable you to easily integrate with a variety of ecommerce backends, payment systems, and other services, allowing you to extend functionality without disrupting the core framework.
+- **Design Components**: Building trust is key in ecommerce. With [Daffodil's component kit](/docs/design), you can streamline development and create a polished, professional store that customers will love.
+- **Performance**: Performance is paramount. Your store can only be performant if our code is fast, you can rely on us to prevent and fix performance issues as quickly as possible.
```

Comment:
```suggestion
- **Performance**: Performance is paramount. Your store can only be performant if our code is fast. You can rely on us to prevent and fix performance issues as quickly as possible.
```